src/App.js

```diff
@@ -10,17 +10,19 @@ import Post from './pages/Post';
 import RandoSug from './pages/RandoSug';
 import Explore from './pages/Explore';
 import DashBoard from './pages/DashBoard';
+import SpongeBob from './components/mockUser.json';
 import Register from './pages/Register';
 
 function App() {
+
   return (
     <Router>
       <Switch>
         <Route path="/explore">
           <Explore />
         </Route>
         <Route path="/profile">
-          <Profile />
+          <Profile data={SpongeBob} />
```

Comment:
This is fine for now. But the profile should source the current user from firebase internally, and parse information from there. In general, the router shouldn't have to pass params that are obtainable from within the component.

Reply:
ok noted i just did for styling our profile page using our mockdata